rocblas/PKGBUILD

```diff
@@ -33,6 +36,8 @@ build() {
                     -Damd_comgr_DIR=/opt/rocm/lib/cmake/amd_comgr
                     -DBUILD_WITH_TENSILE=ON
                     -DTensile_LIBRARY_FORMAT=yaml
+                    -Dtensile_fork="DorianRudolph"
+                    -Dtensile_tag="1eba330320cf798e8a37e3de332ad4480110a430"
```

Comment:
```suggestion
```
These shouldn't be needed anymore. Thanks to https://github.com/ROCmSoftwarePlatform/Tensile/pull/1511 being merged.

Reply:
I see now why this was helpful. We can patch the tag but would also need to patch the cmake requirements and I think that's getting out of scope. On the bright side, if I'm following this correctly https://github.com/ROCmSoftwarePlatform/rocBLAS/commit/81d960138070c9225c3f509952fffefa3d1bdbcc this update to the latest dev branch which includes the merge would mean that 5.4 will support several more architectures. So provided https://github.com/ROCmSoftwarePlatform/rocBLAS/pull/1251 gets merged before then, we won't have to do anything.